doc/source/reference/c-api.coremath.rst

```diff
@@ -81,7 +81,7 @@ Useful math constants
 ~~~~~~~~~~~~~~~~~~~~~
 
 The following math constants are available in npy_math.h. Single and extended
-precision are also available by adding the F and L suffixes respectively.
+precision are also available by adding the f and l suffixes respectively.
```

Comment:
nit: might want to use ``` ``f`` and ``l`` ``` here for clarity

Reply:
Tossed one around `npy_math.h` too for good measure.